quickstarts/mssql-queryplan/config.yml

```diff
@@ -0,0 +1,31 @@
+# Name of the quickstart (required)
+name: nri-mssql-experimental
+title: Microsoft SQL Server with Query Plan logging
+description: |
+  The original SQL Server nri with added Query Plan logging
```

Comment:
Would you be able to add more content to the description? This is a little light on product value and why it's useful to an end user.